### DISCUSSION TITLE
Should we use Crowdin for translations?

### DISCUSSION THREAD

**Discussion by vectorwing** [vectorwing/FarmersDelight#767] [Polls]
**DISCLAIMER: This poll isn't a definitive decision - I'm just gathering opinions on this topic.**

**Crowdin** is a known platform for managing translation efforts for software projects. Open-source projects can set one up for free, and I've seen both Mojang and some Minecraft mods using it for decent success, such as Create and Ars Nouveau.

Here's [Minecraft's official Crowdin page](https://crowdin.com/project/minecraft), as an example.

Farmer's Delight gets a pretty significant traffic for translation efforts - lots of amazing contributors have localized this mod over its three-year lifespan. However, not every potential translator is familiar with Github, or even Git - there is a barrier in regards to messing with JSON formatting and version control. A few translators resort to directly DM-ing me their .json files, for example.

Another point is that I don't have a good way of notifying potential translators that new langs were added. This usually results in PR's arriving after a version release, which is about the only notification I can give without spamming everyone... 😅

I'm still not 100% sure on what it takes to get Crowdin integration, but I'm studying it now and then. It could potentially help with both translating and proofreading, while alleviating the technical parts. As far as I could tell, it would not exclude direct PR's either - just gotta see how that works out.

---

So basically, this thread is to discuss the possibility and details of this integration. Feel free to post below with your opinion, and any information you'd like to chime in about this. What do you folks think? 📚

**Comment by cerealconjugo** [comment]
Seems nice to me, usually i don’t have problems using GitHub, but with mods that are in constant update process, its really easy to lost clues on the new missing strings, i think it will help, deffinetly

**Comment by CKenJa** [comment]
Tralslate by crowdin are also disadvantages related to management difficulties.
For example, difficulty in achieving uniformity among strings and language administrators running amok.
If they are to be introduced, they need to be taken into account.
